l10n_it_ricevute_bancarie/models/riba_config.py

```diff
@@ -24,17 +24,19 @@ class RibaConfiguration(models.Model):
         "res.partner.bank",
         "Bank Account",
         required=True,
+        domain="[('company_id', '=', company_id)]",
         help="Bank account used for C/O issuing.",
     )
     acceptance_journal_id = fields.Many2one(
         "account.journal",
         "Acceptance Journal",
-        domain=[("type", "=", "bank")],
+        domain="[('company_id', '=', company_id), ('type', '=', 'bank')]",
```

Comment:
Dove c'è anche un dominio non riguardante la company, aggiungerei
```suggestion
        domain="[('company_id', '=', company_id), ('type', '=', 'bank')]",
        check_company=True,
```